client/Containers/Main.jsx

```diff
@@ -1,31 +1,22 @@
 import React, { Component } from 'react';
 import UserDashboard from './UserDashboard.jsx';
 import { connect } from 'react-redux';
-import {
-  HashRouter as Router,
-  Route,
-  Switch,
-  Link,
-  useParams,
-} from 'react-router-dom';
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
 import AllProducts from './AllProducts';
 import MainNav from './MainNav';
 import HomePage from '../Containers/HomePage';
 
 class Main extends Component {
   render() {
-    const { fetchProductList } = this.props;
-    console.log('Main', this.props);
     return (
       <>
         <Router>
           <MainNav />
-          <Route component={HomePage} path="/" exact />
-          <div id="container">
-            {/* <Route component={MainNav} /> */}
+          <Switch>
+            <Route component={HomePage} path="/" exact />
             <Route component={AllProducts} path="/products" exact />
             <Route component={UserDashboard} path="/login" exact />
-          </div>
+          </Switch>
```

Comment:
Moved these into a `Switch` so we can add an error page and avoid accidental multiple displays as we add more options here

Reply:
Good catch